lib/src/core/better_player_controller.dart

```diff
@@ -487,7 +487,9 @@ class BetterPlayerController {
                 ?.notificationConfiguration?.notificationChannelName,
             overriddenDuration: _betterPlayerDataSource!.overriddenDuration,
             activityName: _betterPlayerDataSource
-                ?.notificationConfiguration?.activityName);
+                ?.notificationConfiguration?.activityName,
+              clearKey: _betterPlayerDataSource?.drmConfiguration?.clearKey
```

Comment:
I think it's should be in all data sources. Does this work with remote sources?

Reply:
Added for network and memory.